test/snapshots/no-single-promise-in-promise-methods.mjs.md

```diff
@@ -0,0 +1,298 @@
+# Snapshot report for `test/no-single-promise-in-promise-methods.mjs`
+
+The actual snapshot is saved in `no-single-promise-in-promise-methods.mjs.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## invalid(1): await Promise.all([promise])
+
+> Input
+
+    `␊
+      1 | await Promise.all([promise])␊
+    `
+
+> Output
+
+    `␊
+      1 | await promise␊
+    `
+
+> Error 1/1
+
+    `␊
+    > 1 | await Promise.all([promise])␊
+        |       ^^^^^^^^^^^^^^^^^^^^^^ Parameter in \`Promise.all()\` should not be a single element array.␊
```

Comment:
Let's set the error range to `callExpression.arguments[0]`?

Reply:
It's done.